crates/shared/src/price_estimation/buffered.rs

```diff
@@ -91,6 +92,7 @@ where
         token: H160,
     ) -> futures::future::BoxFuture<'_, NativePriceEstimateResult> {
         async move {
+            Metrics::inc_estimator("buffered");
```

Comment:
This would become problematic as soon as there are multiple different APIs that support buffering.

Reply:
Btw if you use the `InstrumentedPriceEstimator` wrapper thingy you don't have to add anything to coingecko. You just wrap it in that struct and it starts instrumenting it.